### DISCUSSION TITLE
Add instead internal module on "hardware" window The UART output

### DISCUSSION THREAD

**Discussion by pfeerick** [EdgeTX/edgetx#1831] [Ideas]
It is will be useful for Radiomaster tx12 users
Many of them use only External module
those users can unsolder internal module and use Bluetooth module for telemetry mirror, for example

Originally posted at #1830 by @andriiyll

**Comment by pfeerick** [comment]
Interesting idea though, I would have thought it was more common for the internal module be replaced with a proper 4in1 or DIY ELRS module. I suppose maybe another option for the internal module could be "UART" where appropriate?

**Reply by andriiyll** [reply]
Also if user use internal module, he can connect UART device via external output
Add opportunity to change type of internal or external module to UART telemetry mirror
